rosdep/python.yaml

```
@@ -6719,6 +6719,15 @@ python3-pyrr-pip:
   ubuntu:
     pip:
       packages: [pyrr]
+python3-pyside2.qtopengl:
+  arch: [pyside2]
+  debian: [python3-pyside2.qtopengl]
```

```suggestion
  debian:
    '*': [python3-pyside2.qtopengl]
    jessie: null
    stretch: null
```

seems to be available from debian buster